server/index.js

```diff
@@ -195,12 +195,12 @@ export default class Server {
         await this.serveStatic(req, res, p)
       },
 
-      '/_next/:buildId/page/:path*': async (req, res, params) => {
+      '/_next/:buildId/page/:path*.js': async (req, res, params) => {
         const paths = params.path || ['']
-        // We need to remove `.js` from the page otherwise it won't work with
-        // page rewrites
-        // eg:- we re-write page/index.js into page.js
-        const page = `/${paths.join('/')}`.replace('.js', '')
+        // URL is asks for ${page}.js (to support loading assets from static dirs)
```

Comment:
Seems like we don't need this comment. The URL pattern speaks for itself.
Good job @timneutkens

Reply:
Done